### PR TITLE
Update default RSpace log level

### PR DESCRIPTION
**Pull request by tgrospic**
## Overview
This small PR is just an update of log template files with default log level for RSpace to be `info`. With current `warn` level RSpace info messages are not displayed as is default for other projects.

Also logging template files are moved to project _resources_ folder.

Default log file in _resources_ is already changed.
https://github.com/rchain/rchain/blob/5693d848699e12458e08092bf146ac00b2f9ef8a/node/src/main/resources/logback.xml#L28

### Please make sure that this PR:
- [x] is at most 200 lines of code (excluding tests),
- [x] meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards),
- [x] includes tests for all added features,
- [x] has a reviewer assigned,
- [x] has [all commits signed](https://rchain.atlassian.net/wiki/spaces/DOC/pages/498630673/How+to+sign+commits+to+rchain+rchain).

### [Bors](https://bors.tech/) cheat-sheet:

- `bors r+` runs integration tests and merges the PR (if it's approved),
- `bors try` runs integration tests for the PR,
- `bors delegate+` enables non-maintainer PR authors to run the above.

### CODE REVIEW

File: node/src/main/resources/logging-template/logback-timeRolling.xml [hunk 1]
```diff
@@ -33,7 +33,7 @@
     </appender>
 
     <logger name="coop.rchain.shared.EventLogger" level="OFF" />
-    <logger name="coop.rchain.rspace" level="warn" />
+    <logger name="coop.rchain.rspace" level="info" />
```
Comment (zsluedem) [inline node/src/main/resources/logging-template/logback-timeRolling.xml:36]: These are only templates.

**Comment by tgrospic** [comment]
@zsluedem I moved templates to resources folder. I'll update the wiki page where they are referenced. Is there some other place to look?

**Review by zsluedem — APPROVED** [review]
I think these are only templates in which the default value are not actually important. : )

**Review by nzpr — APPROVED** [review]
LGTM

**Comment by tgrospic** [comment]
bors merge

**Comment by tgrospic** [comment]
bors ping

**Comment by tgrospic** [comment]
bors ping

**Comment by tgrospic** [comment]
bors merge

**Pull request merged**